lib/oneview-sdk/resource/api300/c7000/switch.rb

```diff
@@ -10,20 +10,20 @@
 # language governing permissions and limitations under the License.
 
 require_relative '../../api200/switch'
+require_relative 'scope'
 
 module OneviewSDK
   module API300
     module C7000
       # Switch resource implementation
       class Switch < OneviewSDK::API200::Switch
+        include OneviewSDK::API300::C7000::Scope::ScopeHelperMethods
```

Comment:
I'm just concerned with the namespace this Mixin is inserted. Maybe this is not the best place to declare it, but I think it's not this PRs concern right now.